packages/xo-server-test/src/issues/index.spec.js

```diff
@@ -115,4 +115,28 @@ describe('issue', () => {
       expect(vm.boot.order).toBe('n' + bootOrder)
     })
   })
+
+  describe('5265',  () => {
+    const rsName = "xo-server-test resource set"
+    const subjects = [
+          'one',
+          'two',
+          'three',
+    ]
+    test('resourceSet.removeSubject call', async () => {
+      const rs = await xo.call('resourceSet.create', {
+        name: rsName,
+        subjects: subjects,
+      })
+
+      await xo.call('resourceSet.removeSubject', {
+        id: rs.id,
+        subject: subjects[0]
+      })
+
+      await xo.call('resourceSet.delete', {
```

Comment:
If the `resourceSet.removeSubject` call fails the resource set will not be deleted, you can use `tempResourceDisposers` to delete this resource automatically at the end of the test. [Example](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server-test/src/_xoConnection.js#L106-L110)